django/contrib/admin/views/main.py

```diff
@@ -103,6 +111,7 @@ def __init__(
             self.page_num = 1
         self.show_all = ALL_VAR in request.GET
         self.is_popup = IS_POPUP_VAR in request.GET
+        self.add_facets = model_admin.show_facets or IS_FACETS_VAR in request.GET
```

Comment:
I just wonder if this should be an enum `{ALWAYS|ALLOW|NEVER}` (or something) — with `NEVER` disabling the faceting entirely. 

(_"One of our admin users brought down production by adding the `IS_FACETS_VAR` query string arg..."_)


🤔

Reply:
Ooooh this is a clever idea, with `NEVER` as the default?

Reply:
I thought `ALLOW` as the default.

Reply:
That's updated, though might not be documented in the clearest of ways. If there is a nice setting based off an enum I can look at, let me know. Happy to update :)